docs/user-interface/images/app-icons.md

```diff
@@ -1,83 +1,215 @@
-<MauiIcon Include="Resources\AppIcon\appicon.svg" Color="#512BD4" />
+<ItemGroup>
+    <MauiIcon Include="Resources\AppIcon\appicon.svg" Color="#FF0000" />
+</ItemGroup>
 ```
 
-<!-- Valid color values are actually derived from the SKColor struct, rather than Microsoft.Maui.Graphics.Colors. -->
-Color values can be specified in hexadecimal, or as a .NET MAUI color. For example, `Color="Red"` is valid.
+Color values can be specified in hexadecimal, using the format: `#RRGGBB` or `#AARRGGBB`. The value of `RR` represents the red channel, `GG` the green channel, `BB` the blue channel, and `AA` the alpha channel. Instead of a hexadecimal value, you may use a named .NET MAUI color, such as `Red` or `PaleVioletRed`.
+
+## Recolor the foreground
 
-At build time, the app icon can be resized to the correct resolutions for the target platform and device. The resulting app icon is then added to your app package.
+If the app icon is composed of a background (`Include`) image and a foreground (`ForegroundFile`) image, the foreground image can be tinted. To tint the foreground image, specify a color with the `TintColor` attribute. The following example tints the foreground image yellow:
+
+```xml
+<ItemGroup>
+    <MauiIcon Include="Resources\AppIcon\appicon.png" ForegroundFile="Resources\AppIcon\appiconfg.svg" TintColor="Yellow" />
+</ItemGroup>
+```
+
+Color values can be specified in hexadecimal, using the format: `#RRGGBB` or `#AARRGGBB`. The value of `RR` represents the red channel, `GG` the green channel, `BB` the blue channel, and `AA` the alpha channel. Instead of a hexadecimal value, you may use a named .NET MAUI color, such as `Red` or `PaleVioletRed`. The named colors are from the <xref:SkiaSharp.SKColors?displayProperty=fullName> type.
+
+## Use a different icon per platform
+
+If you want to use different icon resources or settings per platform, add the `Condition` attribute to the `<MauiIcon>` item, and query for the specific platform. If the condition is met, the `<MauiIcon>` item is processed. Only the first valid `<MauiIcon>` item is used by .NET MAUI, so all conditional items should be declared first, followed by a default `<MauiIcon>` item without a condition. The following XML demonstrates declare a specific icon for Windows and a fallback icon for all other platforms:
+
+```xml
+<ItemGroup>
+    <!-- App icon for Windows -->
+    <MauiIcon Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'windows'"
+              Include="Resources\AppIcon\backicon.png" ForegroundFile="Resources\AppIcon\appiconfg.svg" TintColor="#40FF00FF" />
+
+    <!-- App icon for all other platforms -->
+    <MauiIcon Include="Resources\AppIcon\appicon.png" ForegroundFile="Resources\AppIcon\appiconfg.svg" TintColor="Yellow" />
+</ItemGroup>
+```
+
+You can set the target platform by changing the value compared in the condition to one of the following values:
+
+- `'ios'`
+- `'maccatalyst'`
+- `'android'`
+- `'windows'`
+- `'tizen'`
```

Comment:
Maybe remove Tizen? I always want to be complete, but maybe it then opens us up to issues about not mentioning Tizen in the platform-specific configuration.